### ISSUE TITLE
Ingest Western Americana (pudl0017)

### ISSUE THREAD

**Issue by tpendragon** [pulibrary/figgy#1581]
Notes: MODS for photos, MARC for Albums; WC54 and 55 have album-level MARC and photo-level MODS :-(

Details: https://docs.google.com/spreadsheets/d/1f-c3goE0CC2QJ0Cbk_-JN67nk3fpI9J2Dl37AhGIzos/edit#gid=24
Depends on #1675
Depends on #1674

A lot of these were ingested, but from the MODS records. These should be ripped out and we should use the item-level component metadata in the EADs.

**Comment by tpendragon** [comment]
Files: https://drive.google.com/drive/u/0/folders/0B4Wo5hgOEFY3ZlZGUU5OS3FIbDA

**Comment by tpendragon** [comment]
I think we should split this into an epic. The files are split into sub-sections:

8892958: We can just ingest it.
c0199: We can just ingest this.
c0622: We can just ingest it
c0938: We can just ingest it
c1353: We can ingest it
c1357: We can ingest it
C1398: We can ingest it
C1407: We can ingest it
wc004: We can ingest it
WC054: This needs its own ticket.
WC055: This needs its own ticket.
WC064: This needs its own ticket.

**Comment by tpendragon** [comment]
@jpstroop @joycebcat What does "cxxxx" and "wcxxxx" signify?

**Comment by tpendragon** [comment]
I blew up each sub-section into individual tickets in case some of the albums have complex viewing use cases.

**Comment by joycebcat** [comment]
@tpendragon You mean the "c" and "wc" in front of the numbers?  Those are part of the collection numbers assigned by Rare Books.  I always assumed the "c" was for collection.  I just called 5 RB people to ask for details but I can't get anybody this morning.

**Comment by jpstroop** [comment]
Yeah, I've already just assumed they were different series of accession
numbers. The info should be carried forward, but I don't think there's a
need to parse it in any special way.

-Js

Sent via mobile. Please excuse typos, brevity, etc.
